test/parse.js

```diff
@@ -19,7 +19,12 @@ var Path = require('path');
 // http://mxr.mozilla.org/mozilla-central/source/js/src/tests/ecma_5/JSON/
 
 var dirsPath = Path.resolve(__dirname, 'parse-cases');
-var dirs = FS.readdirSync(dirsPath);
+var dirs = FS.readdirSync(dirsPath).filter(function(fn) {
+  if (fn.startsWith('.')) {
```

Comment:
You can replace line 23 with:
```
  if (fn.substr(0, 1) == '.') {
```
to support older versions of JS. String.substr is close to 20 years old now :)